chart/values.yaml

```diff
@@ -0,0 +1,3071 @@
+    ## E.g.
+    ## values:
+    ##   - e2e-az1
+    ##   - e2e-az2
+    ##
+    values: []
+  ## @param router.affinity Affinity for router pods assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  ## NOTE: `router.podAffinityPreset`, `router.podAntiAffinityPreset`, and `router.nodeAffinityPreset` will be ignored when it's set
+  ##
+  affinity: {}
+  ## @param router.nodeSelector Node labels for router pods assignment
+  ## ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+  ## @param router.tolerations Tolerations for router pods assignment
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+  ## @param router.updateStrategy.type router statefulset strategy type
+  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
+  ##
+  updateStrategy:
+    ## StrategyType
+    ## Can be set to RollingUpdate or OnDelete
+    ##
+    type: RollingUpdate
+  ## @param router.priorityClassName router pods' priorityClassName
+  ##
+  priorityClassName: ""
+  ## @param router.schedulerName Name of the k8s scheduler (other than default) for router pods
+  ## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+  ##
+  schedulerName: ""
+  ## @param router.lifecycleHooks for the router container(s) to automate configuration before or after startup
+  ##
+  lifecycleHooks: {}
+  ## @param router.extraEnvVars Array with extra environment variables to add to router nodes
+  ## e.g:
+  ## extraEnvVars:
+  ##   - name: FOO
+  ##     value: "bar"
+  ##
+  extraEnvVars: []
+  ## @param router.extraEnvVarsCM Name of existing ConfigMap containing extra env vars for router nodes
+  ##
+  extraEnvVarsCM: ""
+  ## @param router.extraEnvVarsSecret Name of existing Secret containing extra env vars for router nodes
+  ##
+  extraEnvVarsSecret: ""
+  ## @param router.extraVolumes Optionally specify extra list of additional volumes for the router pod(s)
+  ##
+  extraVolumes: []
+  ## @param router.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the router container(s)
+  ##
+  extraVolumeMounts: []
+  ## @param router.sidecars Add additional sidecar containers to the router pod(s)
+  ## e.g:
+  ## sidecars:
+  ##   - name: your-image-name
+  ##     image: your-image
+  ##     imagePullPolicy: Always
+  ##     ports:
+  ##       - name: portname
+  ##         containerPort: 1234
+  ##
+  sidecars: {}
+  ## @param router.initContainers Add additional init containers to the router pod(s)
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+  ## e.g:
+  ## initContainers:
+  ##  - name: your-image-name
+  ##    image: your-image
+  ##    imagePullPolicy: Always
+  ##    command: ['sh', '-c', 'echo "hello world"']
+  ##
+  initContainers: {}
+  ## @section router Service Parameters
```

Comment:
Should we include an Ingress Rule (+ TLS termination via TLS secrets) for this Router service?

Reply:
In my opinion, the ingress should substitute the router component (that's what happens in the Harbor chart). I'd leave this as something to analyze with the CARTO team but I'd leave it out of the scope of this first cut.